src/main/io/ledstrip.c

```diff
@@ -1006,11 +1022,13 @@ void ledStripInit(ledConfig_t *ledConfigsToUse, hsvColor_t *colorsToUse, failsaf
     ledStripInitialised = false;
 }
 
+#define ROTATION_SEQUENCE_LED_WIDTH 0x03       // 0x03 = 2 leds ON ; 0x05 = 3 leds ON
```

Comment:
ROTATION_SEQUENCE_LED_MASK makes more sense ... or use `(1 << n) - 1` to get n 1 bits